File: core/feature03_test.js
var feature03Names = ["R", "A"];
var feature03Accuracies = [0, 2];
var feature03Types = [true, true];

function test01Feature03() {
  var arguments = {"K": 200, "G": 300, "D": 100};
  var expected = {"R": 250, "A": 57.30};
  genericTester(feature03_core(arguments), expected, feature03Names, feature03Accuracies, feature03Types);
}

test01Feature03();

File: core/feature03_test.go
package main

import (
	"testing"
)

var (
	feature03Names      = []string{"R", "A"}
	feature03Accuracies = []string{"%.0f", "%.2f"}
	feature03Types      = []bool{true, true}
)

func Test01Feature03(t *testing.T) {
	arguments := map[string]interface{}{"K": 200, "G": 300, "D": 100}
	expected := map[string]interface{}{"R": 250, "A": 57.30}
	genericTester(t, feature03(arguments), expected, feature02Names, feature02Accuracies, feature02Types)
}
